crowbar_framework/config/initializers/filter.rb

```diff
@@ -18,6 +18,9 @@
 Rails.application.config.tap do |config|
   config.filter_parameters += [
     :password,
-    :password_confirmation
+    :password_confirmation,
+    :properties,
+    :proposal_attributes,
+    :secret
```

Comment:
Unfortunately:

> Parameters: {"utf8"=>"✓", "authenticity_token"=>"Ltw91V6&#x2F;7c8i43rctf+mhfcnZdlLY3UT04Dx3jvD2byPXK5nsNcPzymJNU4js2C1OlqOwZwRNxI3pjfllXhNlw==", "barclamp"=>"nova", "name"=>"default", "submit"=>"Save", "proposal_attributes"=>"[FILTERED]", "scheduler_ram_allocation_ratio"=>"1.0", "scheduler_cpu_allocation_ratio"=>"16.0", "use_migration"=>"true", "setup_shared_instance_storage"=>"false", "use_shared_instance_storage"=>"false", "kvm_ksm_enabled"=>"true", "vcenter_host"=>"", "vcenter_user"=>"", "vcenter_password"=>"[FILTERED]", "vcenter_clusters"=>"", "vcenter_datastore"=>"", "vcenter_interface"=>"vmnic0", "vcenter_ca_file"=>"", "vcenter_insecure"=>"false", "zvm_zvm_xcat_server"=>"", "zvm_zvm_xcat_username"=>"", "zvm_zvm_xcat_password"=>"[FILTERED]", "zvm_zvm_diskpool"=>"", "zvm_zvm_diskpool_type"=>"", "zvm_zvm_host"=>"", "zvm_zvm_user_profile"=>"", "zvm_zvm_scsi_pool"=>"", "zvm_zvm_xcat_master"=>"", "zvm_zvm_xcat_ssh_key"=>"", "ssl_enabled"=>"false", "ssl_generate_certs"=>"false", "ssl_certfile"=>"/etc/nova/ssl/certs/signing_cert.pem", "ssl_keyfile"=>"/etc/nova/ssl/private/signing_key.pem", "ssl_insecure"=>"false", "ssl_cert_required"=>"false", "vnc_keymap"=>"en-us", "novnc_ssl_enabled"=>"false", "novnc_ssl_certfile"=>"", "novnc_ssl_keyfile"=>"", "verbose"=>"true", "proposal_deployment"=>"{\"crowbar-revision\":0,\"crowbar-applied\":false,\"schema-revision\":41,\"element_states\":{\"nova-controller\":[\"readying\",\"ready\",\"applying\"],\"nova-compute-docker\":[\"readying\",\"ready\",\"applying\"],\"nova-compute-hyperv\":[\"readying\",\"ready\",\"applying\"],\"nova-compute-kvm\":[\"readying\",\"ready\",\"applying\"],\"nova-compute-qemu\":[\"readying\",\"ready\",\"applying\"],\"nova-compute-vmware\":[\"readying\",\"ready\",\"applying\"],\"nova-compute-xen\":[\"readying\",\"ready\",\"applying\"],\"nova-compute-zvm\":[\"readying\",\"ready\",\"applying\"],\"nova-ha-compute\":[\"readying\",\"ready\",\"applying\"]},\"elements\":{\"nova-controller\":[\"cluster:services\"],\"nova-compute-hyperv\":[],\"nova-compute-kvm\":[\"remotes:services\"],\"nova-compute-qemu\":[],\"nova-compute-xen\":[]},\"element_order\":[[\"nova-controller\"],[\"nova-compute-docker\",\"nova-compute-hyperv\",\"nova-compute-kvm\",\"nova-compute-qemu\",\"nova-compute-vmware\",\"nova-compute-xen\",\"nova-compute-zvm\"]],\"element_run_list_order\":{\"nova-controller\":95,\"nova-compute-docker\":97,\"nova-compute-hyperv\":97,\"nova-compute-kvm\":97,\"nova-compute-qemu\":97,\"nova-compute-vmware\":97,\"nova-compute-xen\":97,\"nova-compute-zvm\":97,\"nova-ha-compute\":97},\"config\":{\"environment\":\"nova-config-default\",\"mode\":\"full\",\"transitions\":false,\"transition_list\":[]},\"crowbar-committing\":false,\"crowbar-status\":\"success\",\"crowbar-failed\":\"\",\"crowbar-queued\":false}", "id"=>"default"}

That's because all entries are part of the form in the barclamp, I guess. So we can filter out the proposal, but the attributes are still there. We can't blindly remove all the parameter logs for all methods, so I see two options: either we remove them just for the actions that save a proposal, or we really build a blacklist of terms based on what the current schemas have (I'm not sure that just password and secret are enough, but I don't know for sure).